core/server/src/main/java/alluxio/worker/AlluxioWorkerService.java

```diff
@@ -0,0 +1,113 @@
+/*
+ * The Alluxio Open Foundation licenses this work under the Apache License, version 2.0
+ * (the "License"). You may not use this work except in compliance with the License, which is
+ * available at www.apache.org/licenses/LICENSE-2.0
+ *
+ * This software is distributed on an "AS IS" basis, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ * either express or implied, as more fully set forth in the License.
+ *
+ * See the NOTICE file distributed with this work for information regarding copyright ownership.
+ */
+
+package alluxio.worker;
+
+import alluxio.Server;
+import alluxio.metrics.MetricsSystem;
+import alluxio.wire.WorkerNetAddress;
+import alluxio.worker.block.BlockWorker;
+import alluxio.worker.file.FileSystemWorker;
+
+import java.net.InetSocketAddress;
+
+/**
+ * A worker in the Alluxio system.
+ */
+public interface AlluxioWorkerService extends Server {
```

Comment:
can we leave a TODO here stating that once we are dropping support for Java 7, we will rename this interface to `AlluxioWorker`